auth/internal/internal.go

```diff
@@ -141,6 +142,24 @@ func GetProjectID(b []byte, override string) string {
 	return v.Project
 }
 
+// DoJSONRequest sends the provided JSON bytes with the client. It reads the response
+// body, checks the status code for errors, and returns the body.
+func DoJSONRequest(ctx context.Context, c *http.Client, url string, bodyBytes []byte, pkg string) ([]byte, error) {
+	req, err := http.NewRequestWithContext(ctx, "POST", url, bytes.NewReader(bodyBytes))
```

Comment:
I think either the name of this function and should be changed, or the signature . This hardcodes that requests flowing through here have to be a POST.

Reply:
Added param `method string` to signature. Please suggest a different name if you want it. The reason for the current name is that it sets Content-Type to application/json.